docs/getting-started/tutorial-sample-filter.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[tutorial-sample-filter]]
+=== Filtering the data
+
+Many visualizations in the *Global Flight* dashboard are interactive. They offer 
+mouse clicks and other controls for you to filter the view of the data. Filters 
+are global&mdash;a change in one visualization simultaneously changes the data 
+in every visualization on the dashboard.
+
+. In the *Controls* visualization, set an *Origin City* and a *Destination City*.
+. Click *Apply changes*.
++
+The `OriginCityName` and the `DestCityName` fields are filtered to match 
+the data you specified. 
++
+For example, this dashboard shows the data for flights from London to Newark
```

Comment:
The results in the screenshot make for an empty screenshot for the dashboard. This is probably pretty minor, but what if we chose origin / destinations that we knew had delay and cancellations?